client/src/components/TitleAndDescription.tsx

```diff
@@ -55,12 +55,17 @@ const styles = {
     }
   `,
   description: css`
+    margin-top: 10px;
     font-size: 16px;
     line-height: 22px;
 
     @media (max-width: 1000px) {
       font-size: 15px;
     }
+
+    .strong {
+      font-weight: 600;
+    }
```

Comment:
@jonidelv I just duplicated this class (from a few lines above) but I think we should find a way to reuse such a generic styles. You are more familiar with emotion, so you might know a good way 😏

Reply:
@gillchristian I have a PR for this